Formula/center-im.rb

```diff
@@ -1,7 +1,7 @@
 class CenterIm < Formula
   desc "Text-mode multi-protocol instant messaging client"
   homepage "https://github.com/petrpavlu/centerim5"
-  url "https://www.centerim.org/download/releases/centerim-4.22.10.tar.gz"
+  url "https://web.archive.org/web/20191105151123/https://www.centerim.org/download/releases/centerim-4.22.10.tar.gz"
```

Comment:
Although there is a v5.0.1 (2017-04-23) available via GitHub, it is using an older `libpurple` so it is not compatible with our `pidgin` formula.